micro-memcached/src/main/java/com/oath/micro/server/elasticache/ConfigureElasticache.java

```diff
@@ -39,24 +39,24 @@ public ConfigureElasticache( @Value("${elasticache.hostname:null}") String hostn
 
 
     @Bean(name = "transientCache")
-    public DistributedCacheManager transientCache() throws IOException, URISyntaxException {
+    public DistributedCache transientCache() {
         try {
             log.info("Creating Memcached Data connection for elasticache cluster: {}", hostname);
-            return new TransientElasticacheDataConnection(createMemcachedClient(), retryAfterSecs, maxRetries);
+            return new ElasticacheConnection(createMemcachedClient(), retryAfterSecs, maxRetries);
```

Comment:
`DistributedCache` and its impl support generics and we are loosing the type info here..